great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -299,8 +299,8 @@ def _build_numeric_columns_rule() -> Rule:
 
         # Step-2: Declare "ParameterBuilder" for every metric of interest.
 
-        column_partition_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_partition_parameter_builder(
-            name="column_values.partition",
+        column_partition_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_histogram_parameter_builder(
```

Comment:
@jcampbell this looks good, but I am noticing code/comment lines duplication (most likely a merge/conflict issue).  Happy to pair to resolve them.  Thanks!

Reply:
I don't see that; can you add to specific places where you're seeing it?